b2sdk/transfer/outbound/upload_manager.py

```diff
@@ -202,46 +202,41 @@ def _upload_small_file(
         content_length = upload_source.get_content_length()
         exception_info_list = []
         progress_listener.set_total_bytes(content_length)
-        with progress_listener:
```

Comment:
I would probably just do `with nullcontext(progress_listener)` just to save myself from this big diff in PR